src/infra/router/health_router.go

```diff
@@ -0,0 +1,13 @@
+package router
+
+import (
+	"go-module/src/interface/controller"
+	"github.com/julienschmidt/httprouter"
+)
+
+func GetHealth() *httprouter.Router {
+	router := httprouter.New()
+	router.GET("/health", controller.CheckHelth)
+
+	return router
+}
```

Comment:
この部分以前のコメントの意図としては、ヘルスチェックのみのルーティングだけにしたほうが良いということです。

今後これだとどんどん他のルーティングも増えていくとfatしてしまいます。

なので
```suggestion
func setHealthRouter(router *httprouter.Router) {
	router.GET("/health", controller.CheckHelth)
}

infra/router.go
func Router() *httprouter.Router {
	router := httprouter.New()
        setHealthRouter(router)
	return router
}
```

でRouter()をmain.goで呼ぶと良いです！

Reply:
別のファイルに切り分けないと、他のルーティングが増えたときに一つのファイルがファットになってしまうということですか？　とりあえず、infra/router.goとinfra/health_router.goに分けました。